### ISSUE TITLE
two argument ListPrimitivelyInstalledOperationsOfCategory  and ListInstalledOperationsOfCategory

### ISSUE THREAD

**Issue by mohamed-barakat** [homalg-project/CAP_project#67]
Is it possible to have per category a list of construction "titles" like

`[ "Kernel", "Cokernel", ...]`

and two argument versions of
- `ListPrimitivelyInstalledOperationsOfCategory`
- `ListInstalledOperationsOfCategory`

where the second argument is a construction title

```
ListInstalledOperationsOfCategory( A, "Kernel" );
[ "KernelObject", "KernelObjectFunctorial",
  "KernelEmbedding", "KernelEmbeddingWithGivenKernelObject",
  "KernelLift", "KernelLiftWithGivenKernelObject" ]
```

**Comment by sebasguts** [comment]
Nevermind, I got it wrong. However, I gave you the possibility to add a string and get all the names containing this string. Do you need more?

**Comment by mohamed-barakat** [comment]
Yes, I would like to see a better restructuring per notion if possible.

**Comment by sebasguts** [comment]
This would mean adding the information to the method record. Why would it be useful? I think the names are really good already.

**Comment by mohamed-barakat** [comment]
It would be nice for the user to see what "Kernel" really means for CAP, i.e., to see exactly those installed operations which are defining for "Kernel".

**Comment by mohamed-barakat** [comment]
Actually I would like to see this independent of any category. Something like:
```
ListDefiningOperations( "Kernel" );
[ "KernelObject", "KernelObjectFunctorial",
  "KernelEmbedding", "KernelEmbeddingWithGivenKernelObject",
  "KernelLift", "KernelLiftWithGivenKernelObject" ]
```

**Comment by sebasguts** [comment]
This is not as easy as it seems. The manual covers most of this, and simply seeing the names will do more harm then it helps. It does not tell you what you have to do, and there are several questions: To which category do isomorphisms belong? Should withgiven methods be displayed. 

I do not see any real use of such a method.

**Comment by sebasguts** [comment]
Extracting the namelist is easy by the way, and so is matching strings. Producing meaningful output for such a method is hard, and I think that's what the manual is for.

**Comment by mohamed-barakat** [comment]
True, this is something which is covered in the manual. I just would like to see a simple interactive command line tool. We can discuss this tomorrow.
